pkg/workflows/wasm/host/module.go

```diff
@@ -434,10 +439,20 @@ func createFetchFn(
 	reader unsafeReaderFunc,
 	writer unsafeWriterFunc,
 	sizeWriter unsafeFixedLengthWriterFunc,
-	modCfg *ModuleConfig, store *store,
+	modCfg *ModuleConfig,
+	store *store,
 ) func(caller *wasmtime.Caller, respptr int32, resplenptr int32, reqptr int32, reqptrlen int32) int32 {
-	const errFetchSfx = "error calling fetch"
+	fetchCounter := int64(0)
```

Comment:
@agparadiso This won't work :/ We need to store the counter in the store, since it should be a request-level counter.